core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/Controller.scala

```diff
@@ -31,21 +31,28 @@ import edu.uci.ics.amber.error.WorkflowRuntimeError
 import scala.concurrent.duration._
 import scala.concurrent.{Await, ExecutionContext}
 
+object ControllerConfig {
```

Comment:
Minor comment: do we want to create another file for the ControllerConfig class?